package.json

```diff
@@ -45,6 +45,7 @@
     "vue-zoomer": "homeday-de/vue-zoomer#0.5.0"
   },
   "devDependencies": {
+    "@babel/helper-compilation-targets": "^7.10.4",
```

Comment:
What is the reason for this package? I didn't get it.

Reply:
It was having this error https://stackoverflow.com/questions/62246824/error-err-package-path-not-exported-no-exports-main-resolved-in-app-node-m

Reply:
When are you getting it? Which command?